index.html

```diff
@@ -0,0 +1,300 @@
+        </ul>
+        <button class="last-btn" type="button">See Project</button>
+      </div>
+    </div>
+    <div class="container-1">
+      <div class="items">
+        <h1>Uber Navigation</h1>
+        <ul>
+          <li>UBER</li>
+          <li>Lead Deveveloper</li>
+          <li>2018</li>
+        </ul>
+        <p>
+          A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer:
+          your car.
+        </p>
+        <ul class="btn-div">
+          <li><a href="" class="btn">HTML</a></li>
+          <li><a href="" class="btn">CSS</a></li>
+          <li><a href="" class="btn ruby">Ruby on rails</a></li>
+          <li><a href="" class="btn">javascript</a></li>
+        </ul>
+        <button class="last-btn" type="button">See Project</button>
+      </div>
+      <img src="images/Snapshoot Portfolio (1).png" />
+    </div>
+  </section>
+  <section class="section-3">
+    <div class="container-3">
+      <div class="about-div">
+        <h1 class="about">About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <p class="social-heading">LET’S CONNECT</p>
+        <ul class="socials">
+          <li>
+            <a href="https://twitter.com/KomezudufasheT"><img src="images/Vector.svg" /> </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/arsenethierry/"><img src="images/Linkedin icon.svg" /></a>
+          </li>
+          <li>
+            <a href="mailto:komezarsene15@gmail.com"><img src="images/Vector (2).svg" /></a>
+          </li>
+          <li>
+            <a href="https://github.com/Arsenethierry"><img src="images/Vector (3).svg" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/Vector (4).svg" /></a>
+          </li>
+        </ul>
+        <button class="last-btn m-b" type="button">Get My Resume</button>
+      </div>
+      <div class="languages">
+        <ul class="languages-div language">
+          <li>Languages</li>
+          <li><img src="images/Union.png" /></li>
+        </ul>
+        <div class="wrapper">
+          <div class="languages-div skills">
+            <img src="images/Ellipse 1.png" />
+            <h5 class="m-l">javascript</h5>
+          </div>
+          <div class="languages-div skills">
+            <img src="images/Ellipse 1 (1).png" />
+            <h5 class="m-l">CSS</h5>
+          </div>
+          <div class="languages-div skills">
+            <img src="images/Ellipse 1 (2).png" />
+            <h5 class="m-l">HTML</h5>
+          </div>
+        </div>
+        <ul class="languages-div framework">
+          <li>Framework</li>
+          <li><img src="images/union 2.png" /></li>
```

Comment:
- [ ] For the Languages section we have to make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li also the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li, like :point_down: 
```
 <ul>
      <li> Language</li>
        <ul>
          <li>Javascript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>Css</li>
        </ul>
```